### ISSUE TITLE
Implementation of scipy.integrate.simps

### ISSUE THREAD

**Issue by Joshuaalbert** [dask/dask#2629]
Are there plans to implement `scipy.integrate.simps`? 
It is a pretty useful method that would easily gain from chunking (when integration is done down one axis of a multi-N array).

**Comment by mrocklin** [comment]
Is this something that you're interested in implementing @Joshuaalbert ?

**Comment by Joshuaalbert** [comment]
Yes, I would like to. Is there a developers guide I should read first?

**Comment by mrocklin** [comment]
I apologize for the long delay here.  http://dask.pydata.org/en/latest/develop.html

**Comment by GenevieveBuckley** [comment]
This issue has been open for a while, contributions are welcome from anyone who'd like to tackle it.

**Comment by mrinalsardar** [comment]
@GenevieveBuckley may I work on this please?
Also maybe we should create a `dask-scipy` sort of repo?

**Comment by wkrasnicki** [comment]
@mrinalsardar @mrocklin @GenevieveBuckley  What is the status of this issue? Can I pick it up?

**Comment by jrbourbeau** [comment]
Based on the content of the issue, it doesn't looks like there has been any progress. Feel free to work on this if you'd like @wkrasnicki, one doesn't need to ask permission to pick up an issue

**Comment by Raahim-Lone** [comment]
Hi everyone, first-time contributor here! I’m tackling this issue right now and will open a draft PR soon. Any feedback is appreciated!

**Comment by Quantum0uasar** [comment]
I'd love to implement this!

Note: `scipy.integrate.simps` has been renamed to `scipy.integrate.simpson` in newer scipy versions. I'll implement `dask.array.simpson` that wraps the scipy implementation to work on chunked arrays via the "overlapping" (overlap/ghost) pattern.

Approach:
- Use `dask.array.lib.stride_tricks` and the overlap pattern to handle chunk boundaries correctly
- Apply Simpson's rule across each chunk with appropriate boundary handling
- Support the `axis` and `dx`/`x` parameters

Would this be best added to `dask/array/routines.py`? Happy to start a draft PR!
